examples/form-callback.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+ <head>
+  <meta charset="UTF-8">
+  <title>Form callback example</title>
+  <script src="/dist/mojeid_connect.2.0.0.js" id="mojeid-connect-script" data-jsrsasign="/node_modules/jsrsasign/lib/jsrsasign-all-min.js"></script>
```

Comment:
In general, maybe we could think about using https://requirejs.org/
It minimalize javascript in html files and also I think that we could use it for jsrslib loading.
But this is to think about impacts, possitives and negatives of using this solution.

Reply:
I wouldn't use it yet. I'd like to keep the project as simple as possible.